### DISCUSSION TITLE
Comfyui Handshake status 403 Forbidden

### DISCUSSION THREAD

**Discussion by pengfan7758258** [open-webui/open-webui#4936] [General]
![4](https://github.com/user-attachments/assets/d1711782-3ed1-4d16-b947-4eaf56e95db5)
![1](https://github.com/user-attachments/assets/d1c5bde9-3029-4d64-8a67-ae175ead91b3)
![2](https://github.com/user-attachments/assets/a5716577-6da5-464a-ad29-d2ba0c837248)
![3](https://github.com/user-attachments/assets/2b49e133-60c7-4080-9188-425ae0d6d68f)
![5](https://github.com/user-attachments/assets/f20e2992-e9fa-4f7b-982c-9de15c70f18a)

**Comment by timtom16** [comment]
I have the exact same issue. Any help would be very much appreciated

**Reply by silentoplayz** [reply]
> I have the exact same issue. Any help would be very much appreciated

A [*possible fix* PR](https://github.com/open-webui/open-webui/pull/5030) for this issue has been opened by a fellow contributor.

**Reply by silentoplayz** [reply]
The PR I had referenced has been merged. Can you update us on if the issue still affects you after updating your Open WebUI instance to the latest version?

**Comment by nielskool** [comment]
i have the same issue as of today.
I had a working setup. a little older open-webui, update to latest version last sunday.
then i could upload workflows and everything was working

Today I updated ollama and completely removed open webui docker container+volume, started a new container with the same latest open_webui image, but i am now getting this same issue.

So it is either the updated ollama or a different setting which is not correctly set in the latest open_webui version. 

I did not change anthing to my comfyui instance

**Comment by nielskool** [comment]
just installed first the v.0.3.10 tag. configured everything, upgraded to main(v.0.3.16) and everything works again on the newest version
